staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/status_strategy.go

```diff
@@ -32,6 +33,25 @@ func NewStatusStrategy(strategy customResourceStrategy) statusStrategy {
 	return statusStrategy{strategy}
 }
 
+// GetResetFields returns the set of fields that get reset by the strategy
+// and should not be modified by the user.
+func (a statusStrategy) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
+	fields := map[fieldpath.APIVersion]*fieldpath.Set{
+		"apiextensions.k8s.io/v1": fieldpath.NewSet(
```

Comment:
i'm not even sure if this is correct.
we are in the customresource strategy, shouldn't the version be the respective CR version and not apiextensions.k8s.io/v1?